.github/workflows/release.yml

```diff
@@ -122,7 +138,9 @@ jobs:
 
       - name: "Prepare release message"
         run: |
-          echo "${{ format(env.default-release-message, env.PACKAGE_VERSION, inputs.jira-version-number) }}" > release-message
+          cat > release-message <<'EOL'
+          ${{ format(env.default-release-message, env.PACKAGE_VERSION, inputs.jira-version-number) }}
+          EOL
```

Comment:
Does this craft a new body for the GitHub release? If so, is there any reason to temporarily store the generated changelog in the earlier "Create draft release with generated changelog" step, or is that necessary because [gh_release_create](https://cli.github.com/manual/gh_release_create) is the only command that actually generates the changelog and can only output it to the release body?

Reply:
Creating a draft release with the auto-generated changelog is the easiest way to get the changelog without using the GitHub API, so that's why I create the draft, extract the changelog and update the draft release in a later step.